gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -98,8 +105,11 @@
             <property name="setterCanReturnItsClass" value="true"/>
         </module>
         <module name="HideUtilityClassConstructor"/> <!-- Java Coding Guidelines: Private constructors -->
+        <module name="IllegalCatch">
```

Comment:
The alternative is to use a comment to mark where this was done intentionally:  `// suppress checkstyle`
But I don't feel strongly - happy to remove.